pkg/defaults/defaults.go

```diff
@@ -48,6 +48,8 @@ const (
 	// ResyncPeriod is the default resync period duration.
 	ResyncPeriod time.Duration = 10 * time.Minute
 
+	Timeout time.Duration = 10 * time.Second
```

Comment:
Same, I don't think we are using any "global" timeout value.

Reply:
I agree, so this would become the default for the shutdown timeout